src/controls/Button.jsx

```diff
@@ -36,59 +37,52 @@ const buttonStyles = {
  * @returns {import("react").ReactComponentElement}
  */
 export default function Button(props) {
-  let {
-    text,
-    onClick,
-    width,
-    type,
-    size,
-    height,
-    hoverStart,
-    backgroundColor,
-    activeBackgroundColor,
-    style,
-  } = props;
+  let { text, onClick, width, type, size, height, style } = props;
 
   // Assign fallback values for styling
   if (!type || !(type in Object.keys(buttonStyles))) {
     type = "default";
   }
 
   return (
-    <HoverBox
-      hoverBackgroundColor={
-        activeBackgroundColor || buttonStyles[type].hoverBackgroundColor
-      }
-      hoverStart={hoverStart || "left"}
+    <AntButton
       style={{
```

Comment:
```suggestion
      style={{
        alignItems: "center"
```

(Unless there's an issue somewhere else in the code with doing this - I haven't found one) Could you add back in the `alignItems` setting? The arrows on the arrow buttons float upward too much if it's removed.